app/Http/Controllers/CarController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\CarRequest;
+use App\Car;
+
+class CarController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Responnse
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+
+    public function index()
+    {
+        $cars = Car::paginate(2);
```

Comment:
i think u should increase to 10

Reply:
Will require a new branch and right now iam doing hot fix branch ,will check through once iam done